app/clients/letter/dvla.py

```diff
@@ -240,7 +240,6 @@ def send_letter(
 
         address should be normalised address lines, e.g. ['A. User', 'London', 'SW1 1AA']
         """
-        current_app.logger.info(f"Sending letter with id {notification_id}")
```

Comment:
I'm not really against still having this line that says "we got to dvla_client.send_letter" 🤔  It's still a useful checkpoint I think.

Reply:
i did pause (because this was how i realised that we had a bug anyway) but i think as far as i can see, by changing the call to raise rather than return, we should never silently exit before that point any more so i think it's less useful now